### PR TITLE
(CU-85zrq9zuh) Centauri content rework and minor fixes

### PR DESCRIPTION
**Pull request by Martin-Kiss**
Additional changes:
- remove codespaces from the table of contents as requested

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [CU-85zrq9zuh Gitbook edits](https://app.clickup.com/t/85zrq9zuh)

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by 0xBrainjar2 — DISMISSED** [review]

File: docs/docs/products/centauri-overview.md [hunk 1]
```diff
@@ -1,39 +1,55 @@
...
-Secondly, Merkle Mountain Ranges (MMRs) allow new block headers to be hashed in an ‘append-only’ manner, 
-which is more suited for use in consensus subsystems other than Merkle trees. Finally, the BEEFY finality gadget makes 
-use of Merkle Mountain Ranges to produce finality proofs for parachains in a more efficient way. 
-These components are highlighted in the sub sections for Centauri.
+## Pallet IBC
 
+Pallet IBC is referred to as the IBC stack for non-Cosmos chains 
+and is composed of IBC-rs and a Tendermint light client(ic07). 
+IBC-rs is an implementation of IBC in Rust, 
+which allows for IBC messages to be interpreted on Picasso(and other Substrate-based chains).
```
Comment (blasrodri) [inline docs/docs/products/centauri-overview.md:24]:
```suggestion
which allows for IBC messages to be interpreted on Picasso (and other Substrate-based chains). 
```

**Review by blasrodri — DISMISSED** [review]

**Comment by JafarAz** [comment]
@mergify rebase

**Comment by josietyleung** [comment]
@JafarAz is this done? thanks!

**Pull request closed without merging**